perfkitbenchmarker/linux_benchmarks/specsfs2014_benchmark.py

```diff
@@ -0,0 +1,264 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs SPEC SFS 2014.
+
+SPEC SFS 2014 homepage: http://www.spec.org/sfs2014/
+
+In order to run this benchmark copy your 'SPECsfs2014_SP1.iso'
+and 'netmist_license_key' files into the data/ directory.
+"""
+
+import posixpath
+import xml.etree.ElementTree
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import data
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker import sample
+
+from perfkitbenchmarker.linux_packages import gluster
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    'specsfs2014_config', None,
+    'This flag can be used to specify an alternate SPEC config file to use. '
+    'If this option is specified, none of the other benchmark specific flags '
+    'which operate on the config file will be used (since the default config '
+    'file will be replaced by this one).')
+flags.DEFINE_enum(
+    'specsfs2014_benchmark', 'VDI', ['VDI', 'DATABASE', 'SWBUILD', 'VDA'],
+    'The SPEC SFS 2014 benchmark to run.')
+flags.DEFINE_integer(
+    'specsfs2014_load', 1,
+    'The starting load in units of SPEC "business metrics". This varies '
+    'from benchmark to benchmark.',
```

Comment:
Updated description:
    'The starting load in units of SPEC "business metrics". The meaning of '
    'business metric varies depending on the SPEC benchmark (e.g. VDI has '
    'load measured in virtual desktops).'